src/core_modules/capture-core/components/WidgetTwoEventWorkspace/WidgetWrapper/WidgetWrapper.container.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import { colors, spacersNum, IconLink16 } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/';
+import type { Props } from './WidgetWrapper.types';
+import { WidgetTwoEventWorkspaceWrapperTypes } from '../index';
+
+const styles = {
+    container: {
+        marginBottom: '16px',
+        margin: '16px',
+    },
+    header: {
+        display: 'flex',
+        alignItems: 'center',
+        paddingBottom: spacersNum.dp16,
+        fontWeight: 500,
+        fontSize: 16,
+        color: colors.grey800,
+    },
+    referalResponse: {
+        padding: spacersNum.dp16,
+        backgroundColor: colors.blue100,
+        borderRadius: '3px',
+    },
+    linkedEvent: {
+        color: colors.blue600,
+        verticalAlign: 'middle',
+        display: 'flex',
+        fontSize: '16px',
+        fontWeight: '500',
+    },
+    icon: {
+        marginRight: spacersNum.dp8,
+    },
+    decription: {
+        margin: `${spacersNum.dp8}px 0`,
+    },
+};
+
+const WidgetWrapperPlain = ({ widget, type, classes }: Props) => {
+    if (type === WidgetTwoEventWorkspaceWrapperTypes.EDIT_EVENT) {
+        return (
+            <div className={classes.container}>
+                <div className={classes.header}> {i18n.t('Refferal response')} </div>
```

Comment:
We've always tried to stay away from using the word referral as there are other potential use-cases for linked stages. Should we switch this to a more generic term?

Edit: based on the design, this actually looks like the stage name of the stage we are currently in. We are adding data to the referral response stage.